jobserver/models/job_request.py

```diff
@@ -40,6 +40,19 @@ def with_started_at(self):
         )
 
 
+class JobRequestManager(models.Manager.from_queryset(JobRequestQuerySet)):
+    use_in_migrations = True
```

Comment:
I'm unsure of the pros/cons of using this manager in migrations. Could you check with a Djangonaut? However, if we do use it in migrations, then we should create a migration, which we should include in the same commit. Run

```sh
python manage.py makemigrations --dry-run
```

to see what we would create.

George have migrations more meaningful names than the defaults. I'd suggest we do the same.

Reply:
I agree with what Iain has suggested.

The `use_in_migrations` option makes the manager available in RunPython operations within a migration ([docs](https://docs.djangoproject.com/en/5.0/topics/migrations/#model-managers)). We don't need to use this manager within a migration right now and don't use RunPython operations that often, but I'd probably go with the principle of least surprise and make it available anyway. Otherwise, it might not be clear to someone in future why it's not available and we probably do want managers to be available in migrations generally.

Reply:
On closer inspection, it seems like something very similar happens in `0007_add_User_order_by_name.py` so I'll press on and add it